docs/src/test/scala/docs/stream/cookbook/RecipeAdhocSource.scala

```diff
@@ -14,11 +14,12 @@
 package docs.stream.cookbook
 
 import java.util.concurrent.atomic.{ AtomicBoolean, AtomicInteger }
-
```

Comment:
Since you have modified the import here, I think keep black line to separate `org. apache. pekko` and `java. util`
besides I think the dependencies of pekko should be on the top part

Reply:
That's right. I can do that later with a sortImports.